### DISCUSSION TITLE
Tab + Name prefix

### DISCUSSION THREAD

**Discussion by sqcred** [Minestom/Minestom#784] [Help]
I want to change the prefix of a player in tab + their name above their head

i have this but it will not show it 
am i doing something wrong?

![carbon (6)](https://user-images.githubusercontent.com/67073397/159136452-52cf30b2-e645-4bdf-8440-ced1ddfa956a.png)

**Comment by emortaldev** [comment]
Does this change the name properly in tab? I know this wont change the name above the player though, I just know setting prefix only worked for me in tab

**Reply by sqcred** [reply]
It also doesnt work in tab for me, but it should change the name above their head i believe

**Reply by emortaldev** [reply]
Oh I believe I use player.setDisplayName() for TAB

**Reply by sqcred** [reply]
also didnt work for me sadly
